### ISSUE TITLE
Streamlink Twitch GUI seems to interfere with InputMapper's exclusive mode

### ISSUE THREAD

**Issue by zapoqx** [streamlink/streamlink-twitch-gui#367]
### Description

This is a very odd behavior I have seen that only first started since I switched from the Livestreamer twitch gui to Streamlink twitch gui.

InputerMapper is a tool used to make PS4 controllers connected to run as an Xbox 360 supported controller.  This adds compatability to games that would otherwise not support the ps4 controller.

To run InputMapper properly, it is best not to have a program running where it takes exclusive access of the controller (example: Uplay launcher, steam, etc) before the exclusive access is launched.  While Streamlink Twitch GUI is active and the controller is not hooked up yet, streamlink stops the control from gaining exclusive mode and instead must be forced to run in shared mode until the program is closed and the controller is reactivated.

### Expected / Actual behavior

Streamlink would have nothing to do with exclusive rights access with another program.

### Reproduction steps

1. Run InputMapper.  (Do not plug in controller yet).
2. Run Streamlink twitch GUI.
3. Activate a PS4 controller (Wireless or USB). Program will return with error and run exclusive mode work around to try and fix the no exclusive rights access.
4. Activate the PS4 controller again.  Program will return with controller in Shared Mode.
5. Disconnect PS4 Controller.
6. Close Streamlink Twitch GUI.
7. Activate the PS4 Controller again.  Program should return with Exclusive Mode.

### Environment details (operating system, etc.)

CPU: Intel i5-4690
OS: Windows 10 Pro 64-bit
RAM: 16 GB
Installed Version & Location: 64-bit installer method of Streamlink Twitch GUI (in Program Files) with Streamlink installed
Ran: Normal (non-Administrator rights mode)

### Comments, logs, screenshots, etc.

Before saying "Shouldn't this be handled by the devs for that app?" come in, while running livestreamer twitch gui (Up to the last version before the name change launch with version 1.0.0), I had no problems running and activating a controller while the program was still running.  I assume something must be running in the backend of the app that has access to the system to block InputMapper's exclusive Mode from running with StreamLink Twitch GUI and I wanted to report this to see if there was a way to not let it interfere without messing with the program's functionality.

**Comment by bastimeyer** [comment]
As you can see in the changelog, along with the name change, the application had a big NW.js upgrade and is now using the latest Chromium version. Chromium is responsible for this exclusive gamepad access, but the gamepad API is not being used in the app code here.

There was a bugfix regarding the HTML5 gamepad API in NW.js [v0.18.5](https://nwjs.io/blog/v0.18.5/) (nwjs/nw.js#5377).
All issues I can find in the [chromium bug tracker](https://crbug.com) are from 2012 and mac related.

The `enable-gamepad-extensions` flag has been added in Chromium 55, but I don't know if NW.js uses this and if this is even related to the issue.
https://stackoverflow.com/questions/40334348/setting-enable-gamepad-extensions-from-chrome-command-line

AFAIK, there are no flags to disable the gamepad API.

You should check if this issue is caused by one of the latest NW.js versions. The archives can be downloaded here: https://dl.nwjs.io
Running nw.exe will just open a blank NW.js window without any app content. The version that's currently used is `v0.19.0` (Chromium 55), the one of the old releases was the legacy version `v0.12.3` (Chromium 41).

Please report the issue on the [NW.js bug tracker](https://github.com/nwjs/nw.js/issues), as this is an upstream issue and unrelated to Streamlink Twitch GUI.
Thanks!

**Comment by zapoqx** [comment]
Thanks for takign the time to respond.  I went ahead and did as you said on testing and first checked the difference by using v0.18.8 first and had the described issue happen.  I went to the one you said was used on the old version of 12.3 and the issue did not happen.  So I went down to v0.17,0 and v0.17.6.  No problems.  v0.18.0, no problems.  So I went through each one of the v.0.18 till I can find the start point that the issue occurred.  v0.18.5 is where it begins which would make it Chromium 54.0.2840.90 where 18.4 was 54.0.2840.71.

I will report the issue to NW.js.  Thanks again!
